apex_ocr/engine.py

```diff
@@ -307,9 +307,17 @@ def process_squad_summary_page(
             elif isinstance(image, Path):
                 pil_image = Image.open(str(image))
                 # Screenshots do not have EXIF data so must resort to file OS stats
-                results_dict["Datetime"] = datetime.fromtimestamp(
-                    image.stat().st_ctime, tz=timezone.utc
-                )
+                linux_screenshot_format = 'Screenshot from %Y-%m-%d %H-%M-%S'
+                windows_screenshot_format = '%Y%m%d%H%M%S_1'
+                
+                try:
+                    screenshot_datetime = datetime.strptime(image.stem, linux_screenshot_format)
+                except ValueError as e:
+                    logger.warning(f"Not linux timestamp {image.stem}")
+                    screenshot_datetime = datetime.strptime(image.stem, windows_screenshot_format)
```

Comment:
This should probably be more in a switch case so we can catch a filename that's neither the linux nor windows default screenshot filename format.